go_covert_lib/client/client_watch.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
A simple script for the frontend that rebuilds the client every time a change is detected. This script is run by higher level build script above, or can be run individually.